openraft/tests/membership/t16_change_membership_cases.rs

```diff
@@ -15,16 +17,46 @@ async fn change_membership_cases() -> anyhow::Result<()> {
     let (_log_guard, ut_span) = init_ut!();
 
     async {
-        change_from_to(btreeset! {0}, btreeset! {1}).await?;
-        change_from_to(btreeset! {0}, btreeset! {1,2}).await?;
-        change_from_to(btreeset! {0}, btreeset! {1,2, 3}).await?;
-        change_from_to(btreeset! {0, 1}, btreeset! {1, 2}).await?;
-        change_from_to(btreeset! {0, 1}, btreeset! {1}).await?;
-        change_from_to(btreeset! {0, 1}, btreeset! {2}).await?;
-        change_from_to(btreeset! {0, 1}, btreeset! {3}).await?;
-        change_from_to(btreeset! {0, 1, 2}, btreeset! {4}).await?;
-        change_from_to(btreeset! {0, 1, 2}, btreeset! {4,5,6}).await?;
-        change_from_to(btreeset! {0, 1, 2, 3, 4}, btreeset! {0, 1, 2, 3}).await?;
+        change_from_to(btreeset! {0}, ChangeMembers::Replace(btreeset! {1})).await?;
+        change_from_to(btreeset! {0}, ChangeMembers::Replace(btreeset! {1,2})).await?;
+        change_from_to(btreeset! {0}, ChangeMembers::Replace(btreeset! {1,2,3})).await?;
+        change_from_to(btreeset! {0, 1}, ChangeMembers::Replace(btreeset! {1,2})).await?;
+        change_from_to(btreeset! {0, 1}, ChangeMembers::Replace(btreeset! {1})).await?;
+        change_from_to(btreeset! {0, 1}, ChangeMembers::Replace(btreeset! {2})).await?;
+        change_from_to(btreeset! {0, 1}, ChangeMembers::Replace(btreeset! {3})).await?;
+        change_from_to(btreeset! {0, 1, 2}, ChangeMembers::Replace(btreeset! {4})).await?;
+        change_from_to(btreeset! {0, 1, 2}, ChangeMembers::Replace(btreeset! {4,5,6})).await?;
+        change_from_to(btreeset! {0, 1, 2, 3, 4}, ChangeMembers::Replace(btreeset! {0,1,2,3})).await?;
+
+        Ok::<(), anyhow::Error>(())
+    }
+    .instrument(ut_span)
+    .await?;
+
+    Ok(())
+}
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 6)]
+async fn add_members_cases() -> anyhow::Result<()> {
+    let (_log_guard, ut_span) = init_ut!();
+
+    async {
+        change_from_to(btreeset! {0}, ChangeMembers::Add(btreeset! {0,1})).await?;
+
+        Ok::<(), anyhow::Error>(())
```

Comment:
There should be more cases for both `Add` and `Remove` members.

E.g., for `add_members()`, there should be at least adding an empty set of nodes, adding existent nodes, adding inexistent nodes, etc.